cpp/src/client/clientChat.cc

```diff
@@ -59,6 +71,14 @@ int clientChat(int sockfd, const std::string &username) {
         }
         case cmd::JOIN: {
           auto chatroom = split(msgToSend, " ", 2)[1].substr(1);
+          //Validate chatroom name
+          auto index = chatroom.find('@');
+          if (index != std::string::npos) {
```

Comment:
``` cpp
if (chatroom.find('@') != std::string::npos) {
```

Reply:
It means msg is not private.

Reply:
yes got that. was saying `index` need not be assigned